README.md

```diff
@@ -3,43 +3,204 @@
+
+This is a general overview of the repo's root directory structure.
+`requirements.txt`, `fabfile.py` and `.circleci` directory contain code for
+installing dependencies, configuring the CircleCI service, and instructions for
+testing and deployment. The repo's root directory also contains
+the `www` folder. Most of the time, contributors will edit or add files in
+the `www` folder.
+
+```bash
+requirements.txt   # file with dependencies used by pip
+.circleci          # directory containing the continuous integration configuration settings
+fabfile.py         # file which is used by continuous integration for testing and deployment
+www                # directory which contains the content of the website
 ├── CodeOfConduct
-├── _assets        # javascript and CSS stuff goes here
-├── _posts         # these are the blog posts written in markdown
-├── _templates     # these are the base templates that other things use.
+├── _assets        # javascript, CSS stuff, and images go here
+├── _posts         # contains blog posts written in markdown
+├── _templates     # contains the base templates (html and Jinja2) used by the site
 ├── about
 ├── archives
 ├── blog
-├── locations
+├── locations      # Use the config.yml file to add new locations or update location info
 ├── resources
 └── sponsor
 ```
 
-## To run locally
+## Setting Up a Development System
 
-**Note**: before continuing, make sure you have headers for Python and libevent
-installed (e.g., on Ubuntu, **python-dev** and **libevent-dev**). Packages in
-requirements.txt require these to compile successfully.
+If you wish to add a location, new chapter, or make code changes, please
+review the next few sections. There are a few tasks to set up a development
+system:
+- [Set up Python2 and a project directory](#set-up-python2-and-a-project-directory)
+- [Create and activate a virtual environment](#create-and-activate-a-virtual-environment)
+- [Fork and clone your pyladies repo](#fork-and-clone-your-pyladies-repo)
+- [Run the site locally](#run-the-site-locally)
+
+### Set up Python2 and a project directory
+
+**Linux, macOS**
+
+1. Check that Python2 is installed with `python2 --version`. If it is not
+   installed, it can be downloaded at https://python.org:
+   ```bash
+   $ python2 --version
+   Python 2.7.16
+   ```
+
+2. (Optional) Learn the directory which this Python 2 version is installed `which python2`:
+   ```bash
+   $ which python2
+   /usr/local/bin/python2
+   ```
+   You may see a different directory name which is fine.
+
+3. Create a directory for development `mkdir pyladies-dev`:
+   ```bash
+   $ mkdir pyladies-dev
+   ```
+
+4. Change into the directory `cd pyladies-dev`:
+   ```bash
+   $ cd pyladies-dev
+
+   # To check your current directory (`<YOUR_PATH>` will be different on
+   # your system.)
+   $ pwd
+   YOUR_PATH/pyladies-dev
+   ```
+
+Great!
+
+**Windows**
+
+The process will be similar though the commands will vary slightly.
```

Comment:
Can we add a link for the commands? For example: https://devblogs.microsoft.com/scripting/table-of-basic-powershell-commands/

Reply:
Sure that works for me.